judo-ui-react/src/main/resources/actor/src/utilities/application/interfaces.ts.hbs

```diff
@@ -3,3 +3,14 @@
 export type AlternativeApplicationGenerator = (translateFn: Function) => Record<string, string>;
 
 export type HandleApplicationChange = (applicationKey: string) => void;
+
+export interface AggregationInputButtonProps {
+  name: string;
+  className: string;
+  disabled?: boolean;
+  onClick?: (event: Event, handleDropdown: (event: Event) => void) => void;
```

Comment:
We don't need `handleDropdown`.

Reply:
[ec0c04e](https://github.com/BlackBeltTechnology/judo-ui-react-template/pull/378/commits/ec0c04ef0c9e729799ce08472123e586dede906b)